spark/spark-base/src/main/java/com/thoughtworks/go/spark/Routes.java

```diff
@@ -259,4 +259,8 @@ public static String name(String name) {
     public class PluginsSPA {
         public static final String BASE = "/admin/plugins";
     }
+
+    public class MetricsConsentSPA {
+        public static final String BASE = "/admin/metrics/consent";
```

Comment:
@jyotisingh -- the current page URL points to `/admin/metrics/consent`, is that ok?

Few options:
`/admin/metrics_collection`
`/admin/metrics_consent`